test/test_rake_task_arguments.rb

```diff
@@ -124,4 +131,10 @@ def test_extra_args_with_less_than_named_arguments
     assert_equal [], ta.extras
   end
 
+  private
+
+  def older_ruby?
```

Comment:
I dropped old ruby support at Rake 11. Could you remove this method and needless condition at L32?